library/HTMLPurifier/HTMLModule/HTML5/Link.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * https://html.spec.whatwg.org/dev/semantics.html#the-link-element
+ */
+class HTMLPurifier_HTMLModule_HTML5_Link extends HTMLPurifier_HTMLModule
+{
+    /**
+     * @type string
+     */
+    public $name = 'HTML5_Link';
+
+    /**
+     * @param HTMLPurifier_Config $config
+     */
+    public function setup($config)
+    {
+        // https://html.spec.whatwg.org/dev/semantics.html#the-link-element
+        $this->addElement('link', 'Flow', 'Empty', null, array(
```

Comment:
Spec says that `<link>` is also a phrasing (inline) element. Add this below declaration:

```php
$this->addElementToContentSet('link', 'Inline');
```